src/libraries/Common/src/Interop/Browser/Interop.Runtime.Bridge.cs

```diff
@@ -0,0 +1,369 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Do you expect that that this is likely to be included into more projects than just `System.Runtime.InteropServices.JavaScript`? 

If not, these internal types should be moved under `System.Runtime.InteropServices.JavaScript/src`.

Reply:
Yes agreed.  The source moving phase should be done in recent commits.